package.json

```diff
@@ -44,20 +49,17 @@
     "deploy:staging:android": "yarn prepare:staging && eas build --platform android --profile staging --auto-submit --non-interactive",
     "deploy:staging:ios": "yarn prepare:staging && eas build --platform ios --profile staging --auto-submit --non-interactive",
     "deploy:staging": "yarn prepare:staging && eas build --platform all --profile staging --auto-submit --non-interactive",
-    "eas-build-pre-install": "base64 --help && echo $ANDROID_FIREBASE_CONFIG | base64 --decode > google-services.json  && cat google-services.json && echo $IOS_FIREBASE_CONFIG | base64 --decode > GoogleService-Info.plist && cat GoogleService-Info.plist",
-    "generate:component": "node ./scripts/create_new_component.js && yarn eslint src --fix && yarn tsc",
+    "eas-build-pre-install": "base64 --help && echo $ANDROID_FIREBASE_CONFIG | base64 --decode > google-services.json && cat google-services.json && echo $IOS_FIREBASE_CONFIG | base64 --decode > GoogleService-Info.plist && cat GoogleService-Info.plist",
     "generate:env:production": "scripts/generate_dotenv.sh production",
```

Comment:
WDYT -> as the future improvement, make all ```generate```  scripts could be within a reach of cli 🤔

Reply:
I was thinking about that, but some scripts have issues with logging to console.

For example when running: `yarn tsc` from node script, don't show errors in console :(

So we need to do it carefully and one by one.